server/routes/posts.js

```diff
@@ -40,6 +40,20 @@ router.post("/", function(req, res) {
     });
 });
 
+router.get("/discovery/:page?", function(req, res) {
+  const postsPerPage = 3; // at 3 for testing Aecio suggests 20
+  const page = req.params.page;
+  const offset = page && page > 0 ? postsPerPage * (page - 1) : 0; // invalid page value return main page
+  Post.find()
+    .sort({ date: "desc" })
+    .skip(offset)
+    .limit(postsPerPage) // e.g. has this before sort
+    .then(result => {
+      res.send(result);
+    })
+    .catch(err => res.status(400).send({ error: err }));
+});
+
```

Comment:
Let's make page a query param instead. So the frontend would call it with `/discovery?page=1` instead of `discovery/1`

Reply:
Also, we should probably index the 'date' field to make .sort() faster